build_projects/shared-build-targets-utils/Utils/HostVersion.cs

```diff
@@ -24,7 +24,8 @@ public class HostVersion : Version
         public string LatestHostPolicyVersionNoSuffix => $"{Major}.{Minor}.{Patch}";
         public string LatestHostPrerelease => ReleaseSuffix;
         public string LatestHostBuildMajor => $"{CommitCountString}";
-        public string LatestHostSuffix => $"{ReleaseSuffix}-{LatestHostBuildMajor}-00";
+        public string LatestHostBuildMinor => "00";
```

Comment:
Why do we even have this part?  Why not just have the commit count and remove `-00` from all usages?